src/commands/synthetics/__tests__/mobile.test.ts

```diff
@@ -0,0 +1,16 @@
+import {promises as fs} from 'fs'
+import * as path from 'path'
+
+import * as mobile from '../mobile'
+
+// tslint:disable-next-line:no-var-requires
+const tmp = require('tmp-promise')
+
+describe('getMD5HashFromFileBuffer', () => {
+  test('correctly compute md5 of a file', async () => {
+    const dir = (await tmp.dir({mode: 0o755, unsafeCleanup: true})).path
```

Comment:
Curious: can't we create a buffer directly instead of writing a file and then getting its buffer?

Reply:
I preferred to not mock because it was easy to reproduce.

Reply:
Humm IMHO if `getMD5HashFromFileBuffer()` required a file path (so `getMD5HashFromFile`), then using `dir` would make more sense! Otherwise, since it's unit testing the mentioned method, passing the buffer directly would be best. Otherwise you are reproducing the code for reading the file in the test 🤔